server/controllers/login/login.js

```diff
@@ -0,0 +1,37 @@
+const { compare } = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const checkUser = require('../../database/queries/checkUser');
+
+exports.post = (request, response, next) => {
+  const { user_name, password } = request.body;
+  let payload;
+  checkUser(user_name)
+    .then((res) => {
+      if (res.rows[0]) {
+        const { user_name, role } = res.rows[0];
+        payload = {
+          user_name,
+          role,
+        };
+        return compare(password, res.rows[0].password);
```

Comment:
why you didn't destructure password?

Reply:
becaause they have the same name from front and database.